### ISSUE TITLE
Accessibility/Screen Recorder: Missing/undetected labels for toolbar items

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#18118]
### Describe the bug

The screen recorder is not detecting the following items in the toolbar (as per the image):
1. Hamburger img
2. Keycloak img
3. Question mark icon

![Screenshot 2022-08-23 at 11 36 37](https://user-images.githubusercontent.com/4890675/186138551-788738aa-be7c-4fc2-9c65-cc34d15e2574.png)

### Version

19.0.0

### Expected behavior

The screen reader reads items in the toolbar.

### Actual behavior

The screen reader doesn't read items in the toolbar.

### How to Reproduce?

1. Have the admin UI running
2. If using Chrome, go to `Edit` >  `Speach` > `Start Speaking`
3. Observe that the items are not being read
